client/odie/context/index.tsx

```diff
@@ -158,6 +160,21 @@ const OdieAssistantProvider = ( {
 		dispatch( recordTracksEvent( event, properties ) );
 	};
 
+	const setMessageLikedStatus = ( message: Message, liked: boolean ) => {
```

Comment:
Noting that we'll need some error handling logic when it's making real requests.